index.php

```diff
@@ -154,10 +154,10 @@ function getinterval()
         </div>
         <div class="box-body">
           <div class="pull-left" style="width:50%">
-            <canvas id="forwardDestinationPieChart" width="120" height="120" class="extratooltipcanvas"></canvas>
+            <canvas id="forwardDestinationPieChart" width="120" height="120" class="extratooltipcanvas no-user-select"></canvas>
           </div>
           <div class="pull-left" style="width:50%">
-            <div id="forward-destinations-legend" class="chart-legend" class="extratooltipcanvas"></div>
+            <div id="forward-destinations-legend" class="chart-legend" class="extratooltipcanvas no-user-select"></div>
```

Comment:
Oops just noticed the duplicate `class`. I'll remove it from here, but I'm not sure if there are more cases.